mne/viz/_brain/_brain.py

```diff
@@ -251,6 +252,14 @@ class Brain(object):
         in different viewing panes.
     surf : str
         FreeSurfer surface mesh name (ie 'white', 'inflated', etc.).
+    rois : list | tuple | dict | str
+        A list of any of the regions of interest in the
+        :ref:`freesurfer lookup table` or a dictionary of rois with
+        their colors as values.
+    aseg : str
+        The anatomical segmentation file. Default ``aseg``. This may be any
+        anatomical segmentation file in the mri subdirectory of the
+        subject directory from the Freesurfer recon-all.
```

Comment:
I would not add new arguments to the constructor. Instead, a new method `brain.add_aseg(...)` (to mirror `brain.add_parcellation`) would be good, e.g.:
```
def add_aseg(aseg='aseg.mgz', labels=None)
```
and by default `labels=None` means "use all labels in the given parcellation"

Reply:
Great thanks, that's why I asked now :)